docs/migration/v4-migration.md

```diff
@@ -37,6 +37,13 @@ A number of changes were made to the configuration options passed to the `Chart`
 * The order of the `ChartMeta` parameters have been changed from `<Element, DatasetElement, Type>` to `<Type, Element, DatasetElement>`.
 
 ### General
+* Chart.js becomes a [ESM-only package](https://nodejs.org/api/esm.html). To use Chart.js, your project should also be a ES module. Make sure to have this in your `package.json`:
```

Comment:
[nit] "a ES module" should be "an ESM module"

Reply:
ESM = ECMAScript Module, so "ESM module" = "ECMAScript Module module"? :)

Reply:
@kurkle there are "**ESM**" and "**ES** module"